packages/block-editor/src/components/block-list/index.js

```diff
@@ -37,7 +37,7 @@ function BlockList( {
 	rootClientId,
 	isDraggable,
 	renderAppender,
-	__experimentalUIParts = {},
+	__experimentalUIParts = { },
```

Comment:
Nitpick:

```suggestion
	__experimentalUIParts = {},
```

so this code will actually not change compared to its current state :slightly_smiling_face: